integration/travis/run_integration.sh

```diff
@@ -2,16 +2,7 @@
 set -ev
 
 export PROJECT_DIR=`pwd`
-
-# Start minimesos
-cd ${PROJECT_DIR}/../travis
-./minimesos up
-
-# Start cook scheduler
-cd ${PROJECT_DIR}/../scheduler
-## on travis, ports on 172.17.0.1 are bindable from the host OS, and are also
-## available for processes inside minimesos containers to connect to
-LIBPROCESS_IP=172.17.0.1 lein run ${PROJECT_DIR}/travis/scheduler_config.edn &
+$PROJECT_DIR/../travis/start_scheduler.sh
```

Comment:
Doesn't `$PROJECT_DIR/../travis/start_scheduler.sh` start a datomic transactor? If so, we don't want that here. We want datomic to run in memory for the integration tests.